libraries/AP_Camera/AP_Camera.cpp

```diff
@@ -55,10 +55,27 @@ const AP_Param::GroupInfo AP_Camera::var_info[] PROGMEM = {
     // @Values: 0:Low,1:High
     // @User: Standard
     AP_GROUPINFO("RELAY_ON",    5, AP_Camera, _relay_on, 1),
-    
+
+    // @Param: MIN_INTERVAL
+    // @DisplayName: Minimum time between photos
+    // @Description: Postpone shooting if previous picture was taken less than preset time ago.
+    // @User: Standard
+    // @Units: milliseconds
+    // @Range: 0 10000
+    AP_GROUPINFO("MIN_INTERVAL",  6, AP_Camera, _min_interval, 0),
+
+    // @Param: MAX_ROLL
+    // @DisplayName: Maximum photo roll angle.
+    // @Description: Postpone shooting if roll is greater than limit.
+    // @User: Standard
+    // @Units: Degrees
+    // @Range: 0 180
+    AP_GROUPINFO("MAX_ROLL",  7, AP_Camera, _max_roll, 45),
```

Comment:
At a minimum you should make the default the maximum value, otherwise there is a behavior change. Realistically the setting a target window smaller then 2 degrees seems kinda crazy anyways.

Regardless of that 180 is still to big for a AP_Int8, either move to a Int16 or 32.